internal/batches/service.go

```diff
@@ -758,7 +742,7 @@ func (svc *Service) FindDirectoriesInRepos(ctx context.Context, fileName string,
 		a.WriteString("query DirectoriesContainingFile() {\n")
 
 		for _, repo := range batch {
-			query := fmt.Sprintf(`file:(^|/)%s$ repo:^%s$ type:path count:99999`, regexp.QuoteMeta(fileName), regexp.QuoteMeta(repo.Name))
+			query := fmt.Sprintf(`file:(^|/)%s$ repo:^%s$ select:file count:99999`, regexp.QuoteMeta(fileName), regexp.QuoteMeta(repo.Name))
```

Comment:
This change should be functionally equivalent. `type:` is problematic because it applies to both an input parameter (e.g., matches files corresponding to the pattern `file:`, or if `file:` were not specified, some pattern) and then also  returns result type `file`. `select:` strictly applies to return type, which is the desired behavior here, so we should prefer that. In time `type:` will be deprecated in favor of more predictable parameters like `select`.